cds/modules/records/bundles.py

```diff
@@ -43,7 +44,7 @@
         'angular': '~1.4.10',
         'angular-sanitize': '~1.4.10',
         'angular-loading-bar': '~0.9.0',
-        'cds': '~0.1.4',
+        'cds': '*',
```

Comment:
could you please add version ``0.2.0`` (is already released)? At the end we have to keep the npm releases for ``cds-js`` because ``dist`` is created during the release and it is cleaner

Reply:
fixed